src/bone.js

```diff
@@ -50,6 +52,12 @@ function compare(attributes, columnMap) {
     }
   }
 
+  for (const columnName in columnMap) {
+    if (!columnNames.has(columnName)) {
+      diff[columnName] = { remove: true };
```

Comment:
`removed`?

Reply:
there is `{ modify: true }` already, so... on the other hand, the properties were added in a way that is more like to tell driver what to update not what happened in model attributes.